components/builder-web/app/sign-in-page/sign-in-page.component.ts

```diff
@@ -37,9 +41,7 @@ export class SignInPageComponent implements OnInit, OnDestroy {
   }
 
   ngOnInit() {
```

Comment:
If this is empty can we just remove the function?

Reply:
Definitely! I'll do that.